test/ibmq/test_ibmq_job.py

```diff
@@ -407,6 +407,16 @@ def test_error_message_qasm(self, qe_token, qe_url):
         message = job_sim.error_message()
         self.assertIn('Job resulted in the following QASM status(es): ', message)
 
+    @requires_qe_access
+    def test_running_job_properties(self, qe_token, qe_url):
+        """Test fetching properties of a running job."""
+        IBMQ.enable_account(qe_token, qe_url)
+        backend = least_busy(IBMQ.backends(simulator=False))
```

Comment:
By convention, when a test performs an execution on a real device, we mark the test as `@slow_test` - mostly as the test is likely to spend a substantial amount of time in the queue.

Can you add the decorator accordingly? It would make the test be skipped by default (both locally and on the CIs), but you can still trigger it manually by using the `QISKIT_TESTS=run_slow` environment variable (it is recommended to use with caution or be prepared to wait, as running all the slow tests might take a while :grin: )